demisto_sdk/commands/validate/tests/validators_test.py

```diff
@@ -1124,31 +1124,33 @@ def test_ignore_test_doc_non_pack_file_should_not_ignore(self, file_path: str):
             - File is not ignored and False is returned
         """
         validate_manager = ValidateManager(check_is_unskipped=False)
-        assert not validate_manager.ignore_test_doc_non_pack_file(file_path)
+        assert not validate_manager.ignore_files_irrelevant_for_validation(file_path)
 
     @pytest.mark.parametrize('file_path',
                              ['Packs/pack_id/Integrations/integration_id/test_data/file.json',
                               'Packs/pack_id/test_data/file.json',
                               'Packs/pack_id/Scripts/script_id/test_data/file.json',
-                              'Packs/pack_id/TestPlaybooks/test_data/file.json'])
-    def test_ignore_test_doc_non_pack_file_test_file(self, file_path: str):
+                              'Packs/pack_id/TestPlaybooks/test_data/file.json',
+                              'Packs/pack_id/pack_metadata.json',
+                              'Packs/pack_id/Integrations/integration_id/command_examples'])
+    def test_ignore_files_irrelevant_for_validation_test_file(self, file_path: str):
         """
         Given
             - File path
         When
-            - File is part of the test_data directory
+            - File is irrelevant for validation
         Then
             - File is ignored and True is returned
         """
         validate_manager = ValidateManager(check_is_unskipped=False)
-        assert validate_manager.ignore_test_doc_non_pack_file(file_path)
+        assert validate_manager.ignore_files_irrelevant_for_validation(file_path)
 
     @pytest.mark.parametrize('file_path',
                              ['OtherDir/Integration/file.json',
                               'TestData/file.json',
                               'TestPlaybooks/file.yml',
```

Comment:
in general, this folder shouldn't be ignored but let's leave it as is (better to deprecate the tests in it).

Reply:
I'm not sure which folder you're talking about